docs/design/autoware-interfaces/ad-api/list/api/route/lanelet/set.md

```diff
@@ -0,0 +1,23 @@
+# /api/route/lanelet/set
+
+- Method: function call
+- Type: [autoware_ad_api_msgs/srv/LaneletRouteSet](../../../../types/autoware_ad_api_msgs/srv/lanelet_route_set.md)
```

Comment:
Seeing the naming convention of `std_srvs` and `nav2`, I think it's better to start with a verb.
http://docs.ros.org/en/api/std_srvs/html/index-msg.html
https://github.com/ros-planning/navigation2/tree/main/nav2_msgs/srv

If you use `LaneletRouteSet`, some people might regard it as `a set of Lanelet Route`.

```suggestion
- Type: [autoware_ad_api_msgs/srv/SetLaneletRoute](../../../../types/autoware_ad_api_msgs/srv/set_lanelet_route.md)
```

Reply:
Renamed https://github.com/autowarefoundation/autoware-documentation/pull/155/commits/e94bb32ea7af7e4f26895fc2e0035ee94f98d845.